hack/util.sh

```diff
@@ -257,6 +257,19 @@ function util::append_client_kubeconfig {
     kubectl config set-context "${client_id}" --cluster="${client_id}" --user="${client_id}" --kubeconfig="${kubeconfig_path}"
 }
 
+# util::append_client_kubeconfig_with_server creates a new context including a cluster and a user to the existed kubeconfig file
+function util::append_client_kubeconfig_with_server {
```

Comment:
Seems the function name not that straightforward.

Reply:
How about `append_existed_kubeconfig_with_new_context_name`?